woocommerce/woocommerce-android
# [Scan to update inventory] Part 3 – Support for manual quantity update

This is **Part 3** of the Scan to update inventory project
Part 1 – https://github.com/woocommerce/woocommerce-android/pull/10250
Part 2 – https://github.com/woocommerce/woocommerce-android/pull/10253
Part 4 – https://github.com/woocommerce/woocommerce-android/pull/10313

This PR allows updating the product's stock quantity manually, by entering the stock quantity value with keyboard.

### Testing instructions

1. Open product list
2. Click "barcode scanner" icon
3. Scan barcode with the SKU of existing product (you can use online barcode generator like [this one](https://barcode.tec-it.com/en/?data=ACGADO))
4. Verify product is found and the bottom sheet is opened
5. Enter quantity manually and click the "update" button
6. Verify product update is successful

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4527432/cc1706cd-a82f-43f0-bd8a-ad6767d54de9
